src/js/angular/guides/controllers.js

```diff
@@ -25,16 +25,27 @@ function GuidesCtrl($scope, $rootScope, GuidesService, $filter, $translate) {
     $scope.init = function () {
         GuidesService.getGuides()
             .then(guides => {
-                $scope.guides = $filter('orderBy')(guides, 'order');
-                $scope.guides.forEach(guide => guide.name = GuideUtils.translateLocalMessage($translate, guide.title));
+                $scope.guides = $filter('orderBy')(guides, 'guideOrder');
+                $scope.guides.forEach(guide => guide.translatedGuideName = GuideUtils.translateLocalMessage($translate, guide.guideName));
                 $scope.matchedElements = $scope.guides;
                 $scope.changePagination();
             });
-    }
 
-    $scope.startGuide = function (guideFileName) {
-        GuidesService.startGuide(guideFileName);
-    }
+        $rootScope.$on('guideResume', () => {
```

Comment:
The controller of “Guides” view will be initialized when the view is opened only then it will be start to listening for the event. This will not work if an user has guide on pause and refresh the page or close browser and open it again on other view and click on button “Guide paused”.

Reply:
I've restored the old mechanism in the service, with the drawback that the guides will be refetched on resume.